datastore/Gemfile

```diff
@@ -15,11 +15,10 @@
 source "https://rubygems.org"
 
 gem "google-cloud-datastore"
-gem "sinatra"
 
```

Comment:
@TheRoyalTnetennba Should `google-style` be required in this file? It is in #489 but not in #542.

Reply:
I kinda like it in the sample Gemfile b/c I run `rubocop -a` a lot to fix hash alignment etc.

Reply:
Ok I'll add it.

Reply:
Added.